api/src/services/__tests__/meetingsService.ts

```diff
@@ -44,7 +44,7 @@ describe('meetingsService', () => {
         [{ id: meetingId, starts_at: startsAt }]
       );
       mockQuery(
-        'select `meeting_notes`.`id` as `id`, `note_text`, `sort_order`, `authoring_participant_id`, `agenda_owning_participant_id` from `meeting_notes` inner join `participants` on `participants`.`id` = `meeting_notes`.`authoring_participant_id` where `participants`.`meeting_id` = ? order by `agenda_owning_participant_id` asc, `sort_order` asc, `meeting_notes`.`created_at` asc',
+        'select `meeting_notes`.`id` as `id`, `note_text`, `sort_order`, `authoring_participant_id`, `agenda_owning_participant_id` from `meeting_notes` inner join `participants` on `participants`.`id` = `meeting_notes`.`authoring_participant_id` where `participants`.`meeting_id` = ? order by `agenda_owning_participant_id is null` asc, `agenda_owning_participant_id` asc, `sort_order` asc, `meeting_notes`.`created_at` asc',
```

Comment:
I just tried to run this query against my database and it failed. How was this tested?

Reply:
The error was:

```
Unknown column 'agenda_owning_participant_id is null' in 'order clause'
```

It looks like the way that it's specified, the whole string `'agenda_owning_participant_id is null'` is quoted as if it's a column name. There must be a different syntax for defining raw order by statements.

Reply:
Thanks for the comment. Will make the appropriate changes.